doc/api/url.md

```diff
@@ -235,6 +235,19 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. 
```

Comment:
Nit: Can you please split lines longer than 80 chars over multiple lines? Thanks!